ccc/ccc.go

```diff
@@ -49,6 +49,9 @@ var sortingRegexp = regexp.MustCompile("_(.+)")
 // Simple handler to handle fatal errors
 func fatal(err error) {
 	_, _ = fmt.Fprintf(os.Stderr, "%s: %s\n", os.Args[0], err)
+	if value, exists := os.LookupEnv("DEBUG_ccc"); exists {
```

Comment:
I'd rather use something like "CONTINUE_ON_ERROR". "DEBUG_ccc" should IMHO print CCC debug messages instead of skipping errors.

Reply:
yes, you are right, i also thought about adding documentation. Let me first check if it works as planned and then i'll add/change both.